content/installation/net/configuration/YAML-template-net.md

```diff
@@ -0,0 +1,359 @@
+<!--
+title: "YAML configuration template for the .NET agent "
+description: "YAML file template for configuring .NET agent properties"
+tags: "installation net agent YAML configuration rules properties"
+-->
+
+Go to the [YAML Properties](installation-netconfig.html#net-yaml) article for more information about this template.
+
+```
+#================================================================================================================================================================================
+#  
+#  Use the properties in this YAML file to configure a Contrast agent. Go to https://docs.contrastsecurity.com/ to determine the order of precedence for configuration values. 
+#  
+#================================================================================================================================================================================
+
+
+#================================================================================
+# Contrast
+# Use the properties in this section to connect the agent to the Contrast UI.
+#================================================================================
+contrast:
+
+# Only set this property if you want to turn off Contrast. Set to `true` to turn the agent on; set to `false` to turn the agent off.
+# enable: true
```

Comment:
It looks like everything in this template file is left aligned. This breaks the yaml format and parsing in the agent. 

The alignment should be unchanged from what is produced by the spec parser.

For example, I'd expect:
`
contrast:
  enable: true
`

Rather than:
`
contrast:
enable: true
`